### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by SatwikaKacham**
created 2 navbars 
1. simple navigation bar
2. eCommerce navigation bar
[CSB](https://yvh3e.csb.app/)

### CODE REVIEW

**Comment by Jay-Gadhiya** [comment]
you forget to close nav and ul in the eCommerce navigation bar code snippet in the link you provide. everything else is perfect.

**Review by Jay-Gadhiya — APPROVED** [review]

File: navigation/navigation.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+</head>
+
+<body>
+
+    <!-- simple navigation bar -->
+    <h2 class="component-heading"> Navigation</h2>
+    <div class="component-description">Navigation is used to navigate to different places in a website in one go. </div>
+    <nav class="nav-container">
+        <div class="nav-logo-and-title">
+            <img class="nav-logo" src="images/logo.png" alt="">
```
Comment (VishalPatil18) [inline navigation/navigation.html]: You could give value to the `alt` attribute as that helps for accessibility

File: navigation/navigation.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+            <div class="nav-title"> CodesUp</div>
+        </div>
+
+        <div class="nav-navigators">
+            <a class="nav-navigator-item" href="#">Home</a>
+            <a class="nav-navigator-item" href="#">Documentation</a>
+        </div>
+
+
+        <button class="nav-github-holder">  <a  href="#"><i class="fab fa-github">  View on Github</i></a></button>
```
Comment (VishalPatil18) [inline navigation/navigation.html]: extra space could be avoided after the `<a` in the `<a>` tag.

File: navigation/navigation.html [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+            <a class="nav-navigator-item" href="#">Kids</a>
+            <a class="nav-navigator-item" href="#">Women</a>
+            <a class="nav-navigator-item" href="#">Men</a>
+
+        </div>
+
+        <div class="nav-search">
+            <span class="material-icons-outlined">
+                    search
+                    </span>
```
Comment (VishalPatil18) [inline navigation/navigation.html:59]: Indentation could be improved. Try to align any opening tag with it's closing tag. That helps in code readability and maintainability.

**Review by VishalPatil18 — APPROVED** [review]
- When we hover on any buttons like home, kids, women, men you gave them a border-bottom, when that happens all the buttons move a bit up. To avoid that you could do is make the buttons `box-sizing: border-box;` and also remove this property. 
```CSS
.nav-navigator-item::after {
    content: "";
}
```
- You could add a border-bottom rather on hover, that would be easy to implement and won't create these kind of bugs
- A few suggestions regarding code indentation and accessibility were added.

> Apart from these the rest looks good. Great work @SatwikaKacham 👍

**Pull request merged**